src/game/header/local.h

```diff
@@ -1015,7 +1015,7 @@ struct edict_s
 	float touch_debounce_time;
 	float pain_debounce_time;
 	float damage_debounce_time;
-	float fly_sound_debounce_time;
+	float fly_sound_debounce_time;	/* now used by insane marines to store pain timeout */
```

Comment:
maybe "now *also* used..." so it's clear that the originally intended usage also still exists?

Reply:
Good call, I'll amend that.

Reply:
I've updated the comment for all 3 PRs now (yq2, rogue and xatrix).